apps/docs/pages/guides/auth/social-login/auth-azure.mdx

```diff
@@ -57,10 +57,7 @@ This will serve as the `client_secret` when you make API calls to authenticate t
 
 This will allow your users to use your custom Azure login page when logging in.
 
-- Select the Directory (Tenant) ID value.
-- The Azure Tenant URL should look like this: `https://login.microsoftonline.com/<tenant-id>`
-
-![Obtain the tenant url](/docs/img/guides/auth-azure/azure-tenant-url.png)
+Set the tenant URL in your Supabase Azure Provider settings to `https://login.microsoftonline.com/common`
```

Comment:
this isn't necessary because the tenant URL defaults to use the `/common` endpoint - https://github.com/supabase/gotrue/blob/25f2dcbc97bac18266f1d3583614656182154f85/internal/api/provider/azure.go#L12

Reply:
yup that sounds good! we can also mention that it defaults to use the `/common` endpoint if it's not set

Reply:
great, I'll update it.

Is there a specific scenario in which the user should use the url with `tenant-id` and not `/common`?

Reply:
sflr, yes, there's a difference between using the tenant-id and common endpoint as mentioned in azure's docs: https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-convert-app-to-be-multi-tenant#update-your-code-to-send-requests-to-common

Reply:
okay, so my understanding is `/<your-tenant-id>` is used when you want only users from a specific Azure organization to be able to sign in? 
For every other use case, we use `/common`

Reply:
yup that's right, in most cases you should be using the /common endpoint unless you want to restrict azure sign-ins to the users added to your tenant

Reply:
thanks. I've updated the docs.

Reply:
this is great, thanks for helping to update this